liota/core/package_manager.py

```diff
@@ -439,10 +451,10 @@ def run(self):
                     self._cmd_handler_list(msg[1])
             elif command == "stat":
                 with package_lock:
-                    if len(msg) != 2:
+                    if len(msg) < 2:
```

Comment:
What about > 2?

Reply:
It was covered inside _cmd_handler_stat()